server/setup_flow.go

```diff
@@ -400,7 +405,7 @@ func (p *Plugin) stepWebhook() flow.Step {
 			Color: flow.ColorPrimary,
 			Dialog: &model.Dialog{
 				Title:            "Jira Webhook URL",
-				IntroductionText: "Please scroll to select the entire URL if necessary.\n\n```{{.WebhookURL}}```\n\nOnce you have entered all options and the webhook URL, select **Create**",
+				IntroductionText: fmt.Sprintf("Please copy and use the link below as webhook URL. Once you have entered all options and the webhook URL, select **Create**. %s", lineBreak) + fmt.Sprintf("%s %s", lineBreak, webhookURL),
```

Comment:
```suggestion
				IntroductionText: fmt.Sprintf("Please copy the link below as the webhook URL. Once you have entered all options and the webhook URL, select **Create**. %s", lineBreak) + fmt.Sprintf("%s %s", lineBreak, webhookURL),
```

Reply:
@Kshitij-Katiyar - The updated text doesn't match the the popup dialog itself. The user only has 2 options: Cancel or Continue. The user isn't able to see any fields to populate.

I recognize that your PR hasn't made changes to that existing text. Are you open to updating your proposed text to provide more guidance to the user who encounters this dialog? If so, what would you recommend? What instructions on this dialog would help guide the user interested in setting up their Jira Webhook?

Reply:
@cwarnermm This dialog is actually instructing the admin to interact with a form in Jira's UI, for installing the integration on that end. There is indeed no indication on the dialog itself, but I believe there is an instruction in a post that opens the dialog.

https://github.com/mattermost/mattermost-plugin-jira/blob/7534a9747b3e53a5f69a74660aaa7bc9f3c00869/server/setup_flow.go#L385-L400

The last line "Select **View Webhook URL**" is referring to the button below the post that opens the dialog

Reply:
Thanks, @mickmister, for this context.
@Kshitij-Katiyar - I approve this PR assuming the suggested change is applied to the dialog text. Thanks!

Reply:
I'm thinking we should specify that the create button is in Jira though. For example:

> Once you have entered all options and the webhook URL, select **Create** on the Jira create webhook page.

Reply:
I agree with your proposal, @mickmister! Thank you! @Kshitij-Katiyar - Please note and update the text as proposed.

Reply:
FYI there's an unresolved request in this thread, to make sure we mention the `Create` button is in Jira's UI

Reply:
@Kshitij-Katiyar - Is this update something you're open to creating a PR for?